website/docs/faqs/connecting-to-two-dbs-not-allowed.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Can I connect my dbt project to two databases?
+---
+
+No. dbt talks to one database at a time, namely your data warehouse.
```

Comment:
Hmmmmm so there is a caveat here.

If someone means two different databases in the same Snowflake account (or projects in a BigQuery account), then the answer could be "yes", as long as the data is queryable from one set of credentials. So you might need to reword here.

Reply:
To add to this ^

I think it would be useful to clarify that while we can't connect to two Snowflake accounts, we can connect to one Snowflake account with multiple databases. I think a key confusion that is creating this is because some people use databases and data warehouses interchangeably, especially because you can connect dbt to Postgres.

Reply:
this is the problem with database being used interchangeable with database management system. The two are certainly not the same, but end up being in conversation. I'll make edits shortly. Thanks